airflow/sensors/base.py

```diff
@@ -344,6 +344,37 @@ def _get_next_poke_interval(
         if not self.exponential_backoff:
             return self.poke_interval
 
+        if self.reschedule:
+            # Calculate elapsed time since the sensor started
+            elapsed_time = run_duration()
+
+            # Initialize variables for the simulation
+            cumulative_time = 0
+            estimated_poke_count = 0
+
+            while cumulative_time < elapsed_time:
+                estimated_poke_count += 1
+                # Calculate min_backoff for the current try number
+                min_backoff = max(int(self.poke_interval * (2 ** (estimated_poke_count - 1))), 1)
+
+                # Calculate the jitter
```

Comment:
```suggestion
                # Calculate the jitter, which is used to prevent multiple sensors simultaneously poking
```

Reply:
included the comment suggestion in the latest commit, Thanks